float32 distance
string name
float32 certainty

int8 region

int8 ROBOT=1
int8 SCREEN=0
int8 OUTSIDE=2
